wageningen2021.html

```diff
@@ -491,6 +491,35 @@ <h2>Importing</h2>
 </section>
 
 
+<section>
+    <h2>Tap directly into FAIR data warehouses</h2>
+    <h4>GRASS GIS understands netCDF data that follows the CF-convention</h4>
+    <div class="container">
+        <div class="col">
+        <ul>
+             <li><b>m.crawl.thredds</b> - crawls data ware houses (thredds server) for netCDF datasets</li>
```

Comment:
Can any type of data be stored in netCDF? I'm trying to grasp why using m. for the module name... Also, would you provide a link to this thredds server?

Reply:
In principle, also point data can be stored in netCDF, amybe also other vector data, but I am not sure.
However, the point is that `m.crawl.thredds` just returns a list of URLs to datasets on a Thredds server, that fulfill search/filter criteria. So it does not handle an spatial data at all...